<ui version="4.0" >
 <class>CNemoSystemOverviewGuiComp</class>
 <widget class="QWidget" name="CNemoSystemOverviewGuiComp" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="LeftFrame" >
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>2</number>
         </property>
         <item>
          <widget class="QFrame" name="frame" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>7</hsizetype>
             <vsizetype>4</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape" >
            <enum>QFrame::Panel</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth" >
            <number>1</number>
           </property>
           <property name="midLineWidth" >
            <number>0</number>
           </property>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>2</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>4</hsizetype>
                <vsizetype>5</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="pixmap" >
               <pixmap>../../../Products/Algorithmica/Nemo/Install/Resources/Icons/SensorsPanel.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="SystemState" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>4</hsizetype>
                <vsizetype>5</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font" >
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text" >
               <string>System State</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="SystemStateFrame" >
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="title" >
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>2</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_2" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>7</hsizetype>
             <vsizetype>4</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape" >
            <enum>QFrame::Panel</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth" >
            <number>1</number>
           </property>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>2</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>4</hsizetype>
                <vsizetype>5</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="pixmap" >
               <pixmap>../../../Products/Algorithmica/Nemo/Install/Resources/Icons/lamp_32.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="sensortitle" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>4</hsizetype>
                <vsizetype>5</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font" >
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text" >
               <string>Sensors</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="SensorTreeFrame" >
           <property name="minimumSize" >
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="title" >
            <string/>
           </property>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>9</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QFrame" name="SensorListFrame" >
              <property name="frameShape" >
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow" >
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" >
               <property name="margin" >
                <number>0</number>
               </property>
               <property name="spacing" >
                <number>6</number>
               </property>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="RightFrame" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>5</hsizetype>
        <vsizetype>5</vsizetype>
        <horstretch>4</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>2</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_3" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>7</hsizetype>
             <vsizetype>4</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape" >
            <enum>QFrame::Panel</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth" >
            <number>1</number>
           </property>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>2</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>4</hsizetype>
                <vsizetype>5</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="pixmap" >
               <pixmap>../../../Products/Algorithmica/Nemo/Install/Resources/Icons/statistic_32.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="SystemState_3" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>4</hsizetype>
                <vsizetype>5</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font" >
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text" >
               <string>Overview</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="SystemOverviewFrame" >
           <property name="title" >
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>2</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_4" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>7</hsizetype>
             <vsizetype>4</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape" >
            <enum>QFrame::Panel</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth" >
            <number>1</number>
           </property>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>2</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>4</hsizetype>
                <vsizetype>5</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="pixmap" >
               <pixmap>../../../Products/Algorithmica/Nemo/Install/Resources/Icons/statistic_32.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="SystemState_4" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>4</hsizetype>
                <vsizetype>5</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font" >
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text" >
               <string>Sensor Data</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="SensorDataFrame" >
           <property name="title" >
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
